src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediatePortalStatement.java

```diff
@@ -94,7 +94,7 @@ public DescribeMetadata describe() {
         proxy. As such, we send the user a descriptive message to help them understand the issue
         in case they misuse the method.
       */
-      logger.log(Level.SEVERE, e.toString());
+      logger.log(Level.SEVERE, e, e::toString);
```

Comment:
What's this change intending to do? AFAIK the string message should always be second parameter in the logger.log() method.

Reply:
Okay Thanks for the info